src/EmployerPR/SFA.DAS.EmployerPR.Api/AppStart/AddConfigurationOptionsExtension.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using SFA.DAS.SharedOuterApi.Configuration;
+
+namespace SFA.DAS.EmployerPR.Api.AppStart;
+public static class AddConfigurationOptionsExtension
+{
+    public static IServiceCollection AddConfigurationOptions(this IServiceCollection services, IConfiguration configuration)
```

Comment:
This is a Service Collection extension so technically can be added to the other class.